assets/js/AddNotificationButtonClosed.js

```diff
@@ -45,6 +47,20 @@ class AddNotificationButtonClosed extends React.Component {
     this.setState({showButton: false});
   }
 
+  handleClick() {
```

Comment:
I'd already forgotten what this logic does and had to read it to figure out -- would be useful to have a short comment like "Add notification to the first module that adds data."